### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Nineeeeeeeee** [ColeTimlin/TikTokAccountStatus#1]
Hey when I run this it works sometimes and other times it gives me the error "Expecting value: line 1 column 1 (char 0)" then crashes the CMD

**Comment by ColeTimlin** [comment]
Do you have all the required modules installed??

**Comment by Nineeeeeeeee** [comment]
I do believe so

**Comment by Nineeeeeeeee** [comment]
Do you have somewhere i can contact you?

**Comment by ColeTimlin** [comment]
Yes, I have Discord, @Abstract#1014. I may not be active so much on there but once I get out of school I will check out this issue

**Comment by Nineeeeeeeee** [comment]
Unfortunately your discord didn’t work, perhaps your discrim changed. And thank you! 

Please could you add my discord - '9'#9999

**Comment by Murrisk** [comment]
> Hey when I run this it works sometimes and other times it gives me the error "Expecting value: line 1 column 1 (char 0)" then crashes the CMD

thats the ratelimit. not much you can do unless you want to use proxies

**Comment by Nineeeeeeeee** [comment]
> > Hey when I run this it works sometimes and other times it gives me the error "Expecting value: line 1 column 1 (char 0)" then crashes the CMD
> 
> thats the ratelimit. not much you can do unless you want to use proxies

It isn't a rate limit because it happens when I use it once sometimes :P While im not rate limited
